corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditorMain.java

```diff
@@ -60,6 +63,8 @@
         + "--truststore=<truststore_file> TrustStore File\n"
         + "--truststore_password=<truststore_password> Truststore Password\n"
         + "--diskPath=<pathToTempDirForLargeTables> Path to Temp Dir\n"
+        + "--epoch=<epoch> Epoch of snapshot\n"
```

Comment:
Is the epoch really needed? I believe whichever epoch you put will work as we base it on the Timestamp instead.. that would make things easier for users..